php/class-customize-concurrency.php

```diff
@@ -193,6 +207,7 @@ public function customize_register( \WP_Customize_Manager $wp_customize ) {
 			return $validity;
 		};
 
+		// @todo fix $wp_customize->settings() is giving a 500/502 when called inside customize_snapshot_save_before
 		foreach ( $wp_customize->settings() as $setting ) {
```

Comment:
It looks like maybe settings() has not been initialized here. I could use unsanitized_post_values() to get the list of setting ids though. Are there any concerns with that approach?